### PR TITLE
feat(dev-portal): INFRA-263 Send password on email when registering service user

### PR DESCRIPTION
**Pull request by SavelevMatthew**

### CODE REVIEW

File: apps/condo/domains/user/schema/RegisterNewServiceUserService.test.js [hunk 1]
```diff
@@ -10,59 +10,68 @@ const { GET_MY_USERINFO } = require('@condo/domains/user/gql')
...
-        const email = userAttrs.email
-        // TODO(DOMA-3146): use the GQLError util
-        await catchErrorFrom(async () => {
-            await registerNewServiceUserByTestClient(admin, { email })
-        }, ({ errors }) => {
-            expect(errors[0].originalError.errors[0].data.messages[0]).toEqual(
-                expect.stringContaining(EMAIL_ALREADY_REGISTERED_ERROR))
+    describe('Logic tests', () => {
+        test('can not register service user with existed email', async () => {
+            const admin = await makeLoggedInAdminClient()
```
Comment (toplenboren) [inline apps/condo/domains/user/schema/RegisterNewServiceUserService.test.js:60]: Best to move admin creation to describe

File: apps/condo/lang/ru/messages/SERVICE_USER_CREATED/email.njk [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Сервисная учетная запись для данного email успешно создана!
+
+Стенд: {{ message.meta.data.serverUrl }}
+Email: {{ message.meta.data.email }}
```
Comment (toplenboren) [inline apps/condo/lang/ru/messages/SERVICE_USER_CREATED/email.njk:4]: Why not 👉 ЭЛЕКТРОПОЧТА 👈  🤜?🤛

**Review by toplenboren — APPROVED** [review]

File: apps/condo/domains/user/schema/RegisterNewServiceUserService.test.js [hunk 3]
```diff
@@ -10,59 +12,72 @@ const { GET_MY_USERINFO } = require('@condo/domains/user/gql')
...
-            await registerNewServiceUserByTestClient(admin, { email })
-        }, ({ errors }) => {
-            expect(errors[0].originalError.errors[0].data.messages[0]).toEqual(
-                expect.stringContaining(EMAIL_ALREADY_REGISTERED_ERROR))
+    describe('Logic tests', () => {
+        test('can not register service user with existed email', async () => {
+            const [, userAttrs] = await registerNewServiceUserByTestClient(admin)
+            const email = userAttrs.email
+            // TODO(DOMA-3146): use the GQLError util
+            await catchErrorFrom(async () => {
```
Comment (Alllex202) [inline apps/condo/domains/user/schema/RegisterNewServiceUserService.test.js:71]: Maybe fix this? :)

**Review by Alllex202 — APPROVED** [review]

File: apps/condo/domains/user/schema/RegisterNewServiceUserService.test.js (hunk 3 above)
Comment (SavelevMatthew) [reply apps/condo/domains/user/schema/RegisterNewServiceUserService.test.js:71]: It's not throwing GLQError as far as I know

**Pull request merged**